staging/src/k8s.io/csi-translation-lib/translate.go

```diff
@@ -32,12 +33,13 @@ var (
 	}
 )
 
-// TranslateInTreeStorageClassParametersToCSI takes in-tree storage class
-// parameters and translates them to a set of parameters consumable by CSI plugin
-func TranslateInTreeStorageClassParametersToCSI(inTreePluginName string, scParameters map[string]string) (map[string]string, error) {
+// TranslateInTreeStorageClassToCSI takes in-tree volume options
```

Comment:
nit: can you change the comment around "in-tree volume options" to mention "in-tree storage class"?